drafts/draft-weis-ippm-ioam-eth.xml

```diff
@@ -293,8 +293,10 @@
       traverses a particular network domain. The term "in-situ" refers to the
       fact that the IOAM data fields are added to the data packets rather than
       being sent within packets specifically dedicated to OAM. This document
-      defines how IOAM data fields are carried as part of encapsulations where
-      the IOAM data follows a header that uses an EtherType to denote the next
+      proposes a new Ethertype for IOAM and defines how IOAM data fields are 
+      carried as part of encapsulations where
+      the IOAM data follows an encapsulation header that uses an EtherType to 
```

Comment:
s/IOAM data/IOAM data fields/

Reply:
I have this updated in the recent changes